draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1749,9 +1755,6 @@ Practical user agent implementations have limits on the number and size of
 cookies that they can store. General-use user agents SHOULD provide each of the
```

Comment:
Just to be explicit, maybe add a sentence mentioning that the user agent MAY limit the max number of cookies they store, and may evict any cookie at any time (whether on orders from the user or due to implementation limitations). And also explicitly say that a limit on the max number of cookies also limits the total size of the stored cookies, due to the length limits which must be enforced in the parsing step.

Reply:
good suggestion, thx.